src/main.jsx

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
+import "./App.scss";
+import Navbar from "./components/Navbar/Navbar";
```

Comment:
Add a new line between the package imports and css, as well as between css and components/pages